.github/workflows/linkcheck.yml

```diff
@@ -0,0 +1,15 @@
+name: Link-check
+on: [push]
```

Comment:
Is is possible to put this:
`workflow_dispatch:` instead push?
I want to be manually for now.